infrastructure/lib/kit-infrastructure.ts

```diff
@@ -15,6 +15,9 @@ export class KITInfrastructure extends Stack {
     const repoUrl = this.getContextOrDefault('FluxRepoURL', "https://github.com/awslabs/kubernetes-iteration-toolkit")
     const repoBranch = this.getContextOrDefault('FluxRepoBranch', 'main')
     const repoPath = this.getContextOrDefault('FluxRepoPath', './infrastructure/k8s-config/clusters/kit-infrastructure')
+    const installEBSCSIDriverAddon = Boolean(this.getContextOrDefault("AWSEBSCSIDriverAddon", "true"))
```

Comment:
If you change this method to be `getContextOrDefault(key: string, def: string | boolean | null)` it should return the context passed and have an if condition like this - 
`if (installKitAddon == true)`

Reply:
>If you change this method to be getContextOrDefault(key: string, def: string | boolean | null) it should return the context passed

It already returns the context passed with the change i made, no need to change ^^^ that function :D. Unless you are implying something here ?

>if (installKitAddon == true)

- hmmm ^^^  setting this value makes the value comparison to be explicit instead of implicit.

- IOW,  this change will ensure  any value that is set through context is taken as `false` unless it is set to `true` `explicitly`(opposite of default typescript behavior for boolean/string conversion). Instead of just empty value is taken as `false` and any other value is taken as `true`( which is default typescript behavior).